cmd/argo/lint/formatter_pretty_test.go

```diff
@@ -1,3 +1,4 @@
+// disabled on windows because colorized output is not supported - go-color will return just the text without colors.
 //go:build !windows
```

Comment:
Could you change these to a `t.Skip` in `TestMain` instead (as mentioned in https://github.com/argoproj/argo-workflows/pull/12011#discussion_r1369420886)?